scripts/webhook-deployment.sh

```diff
@@ -1,10 +1,24 @@
 #!/bin/bash
 
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 # Always exit on errors.
 set -e
 
 # Set our known directories and parameters.
-BASE_DIR=$(cd $(dirname $0)/..; pwd)
+BASE_DIR="$(cd "$(dirname "$0")"/..; pwd)"
```

Comment:
we have double quotes inside double quotes, do we need escape? or should it just work?

Reply:
It works fine this way with bash. I have tested it. You can have nested double quotes when using $().